autogpt_platform/backend/test/conftest.py

```diff
@@ -12,7 +12,7 @@ async def server():
 @pytest.fixture(scope="session", autouse=True)
 async def graph_cleanup(server):
     created_graph_ids = []
-    original_create_graph = server.agent_server.create_graph
+    original_create_graph = server.agent_server.test_create_graph
```

Comment:
This fixture wraps `create_graph` so that you will clean up all the created graph post-tests.
So the code below it.

If you move everything to test_create_graph, then each test execution will keep piling more graphs without any cleanups.

Reply:
I've changed it to wrap `test_create_graph` instead.

The reason for this, is to allow these functions to still be accessable via the AppSever class, which is how the tests access this functionality.

I renamed them test to clearly denote they are not used during runtime.